services/web/server/src/simcore_service_webserver/login/_2fa.py

```diff
@@ -92,8 +93,9 @@ def sender():
         client = Client(twilo_auth.TWILIO_ACCOUNT_SID, twilo_auth.TWILIO_AUTH_TOKEN)
         message = client.messages.create(
             messaging_service_sid=twilio_messaging_sid,
+            from_=product_display_name,
             to=phone_number,
-            body=f"Dear {product_display_name} user, your verification code is {code}",
+            body=f"Dear {user_name[:20].capitalize().strip()}, your verification code is {code}",
```

Comment:
why 20?

Reply:
I wanted to limit the size of the SMS. I looked in the database and the largest name was around 15 characters.